app/assets/javascripts/discourse/app/mixins/card-contents-base.js

```diff
@@ -117,20 +122,22 @@ export default Mixin.create({
         return true;
       });
 
-    $("#main-outlet").on(clickDataExpand, `[data-${id}]`, (e) => {
-      if (wantsNewWindow(e)) {
-        return;
-      }
-      const $target = $(e.currentTarget);
-      return this._show($target.data(id), $target);
-    });
+    _cardClickListenerSelectors.forEach((selector) => {
```

Comment:
One thing worth a try is joining the selectors with a comma. Then you wouldn't have to loop!

For example: `#main-outlet, #chat-window`